packages/webpack-scaffold/tsconfig.json

```diff
@@ -1,3 +1,3 @@
-{
-  "extends": "../../tsconfig.packages.json"
-}
+{
```

Comment:
If you can, would you be able to revert these changes that don't really contain any real change?

Reply:
Will try 👍

Reply:
I updated line feed from `LF` to `CRLF` but no luck